webapp/plugins/vue-filters.js

```diff
@@ -39,15 +38,6 @@ export default ({ app = {} }) => {
       }
       return accounting.formatNumber(value || 0, precision, thousands, decimals)
     },
-    // format seconds or milliseconds to durations HH:mm:ss
-    duration: (value, unit = 's') => {
-      if (unit === 'ms') {
-        value = value / 1000
-      }
-      return value
-        ? format(addSeconds(new Date('2000-01-01 00:00'), value), 'HH:mm:ss')
-        : '00:00:00'
-    },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-03T18:47:18Z" title="Wednesday, July 3rd 2019, 8:47:18 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -39,15 +38,6 @@ export default ({ app = {} }) => {
       }
       return accounting.formatNumber(value || 0, precision, thousands, decimals)
     },
-    // format seconds or milliseconds to durations HH:mm:ss
-    duration: (value, unit = 's') => {
-      if (unit === 'ms') {
-        value = value / 1000
-      }
-      return value
-        ? format(addSeconds(new Date('2000-01-01 00:00'), value), 'HH:mm:ss')
-        : '00:00:00'
-    },
```


nice that you figured out we don't use this, removing unused dependencies should be an ongoing task